README.md

```diff
@@ -3,6 +3,8 @@
 [![GoDoc](https://godoc.org/github.com/signalfx/signalfx-agent?status.svg)](https://godoc.org/github.com/signalfx/signalfx-agent)
 [![CircleCI](https://circleci.com/gh/signalfx/signalfx-agent.svg?style=shield)](https://circleci.com/gh/signalfx/signalfx-agent)
 
+SignalFx Smart Agent is deprecated. For details, see the [Deprecation Notice](/docs/smartagent-deprecation-notice).
```

Comment:
Don't you think this should have some bolding to make it stand out a bit more?

Same in all the other places.

Reply:
It's first line of the topic, right under the title. Not vivid enough?

Reply:
More vivid the better see: https://github.com/signalfx/gateway#deprecation-notice. In fact, would be best to have everything in doc in main readme and all links point to main readme.